src/assets/translations/messages-en.json

```diff
@@ -61,6 +61,57 @@
       "verify": "Verify"
     }
   },
+  "wProvider": {
+    "keepKey": {
+      "header": "Enter Your Pin",
+      "body": "Use PIN layout shown on your device to find the location to press on the pin pad.",
+      "button": "Unlock"
+    },
+    "shapeShift": {
+      "nImport": {
```

Comment:
what does the `n` prefix of these keys denote? don't be afraid to be explicit - i'm not sure what this means.

Reply:
'Native' - just abbreviating for sake of typing out the variables